python/cac_tripplanner/cac_tripplanner/settings.py

```diff
@@ -0,0 +1,96 @@
+"""
+Django settings for cac_tripplanner project.
```

Comment:
FWIW, we've been trying to pull everything in the Django settings file from the environment and then launch the Django application using `envdir`. Adds some noise before certain commands, but allows us to commit the settings files into the repository.